<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Fenetre_recherche</class>
 <widget class="QWidget" name="Fenetre_recherche">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>441</width>
    <height>185</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QVBoxLayout" name="global">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Rechercher un mot de passe</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>IP de l'interface de configuration</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_ip"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>ou/et Nom du mat√©riel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edit_nom"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="bouton_valider">
         <property name="text">
          <string>Valider</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
